icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java

```diff
@@ -339,6 +339,13 @@ public static CurrencyPrecision currency(CurrencyUsage currencyUsage) {
         }
     }
 
+    /**
+     * Configure how trailing zeros are displayed on numbers. For example, to hide trailing zeros
+     * when the number is an integer, use HIDE_IF_WHOLE.
+     *
+     * @param trailingZeroDisplay Option to configure the display of trailing zeros.
+     * @draft ICU 69
```

Comment:
`@provisional`?

Reply:
We no longer use @provisional tag. See https://unicode-org.atlassian.net/browse/ICU-21369